src/strands/multiagent/a2a/server.py

```diff
@@ -40,13 +43,34 @@ def __init__(
             agent: The Strands Agent to wrap with A2A compatibility.
             host: The hostname or IP address to bind the A2A server to. Defaults to "0.0.0.0".
             port: The port to bind the A2A server to. Defaults to 9000.
+            http_url: The public HTTP URL where this agent will be accessible. If provided,
```

Comment:
As a follow up can we also mention this new introduced parameters in https://strandsagents.com/latest/documentation/docs/user-guide/concepts/multi-agent/agent-to-agent/ ?